sdk/eventhub/azure-eventhub/stress/templates/network_loss.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: chaos-mesh.org/v1alpha1
+kind: NetworkChaos
+metadata:
+  name: '{{ $.Release.Name }}-{{ $.Release.Revision }}'
+  namespace: {{ $.Release.Namespace }}
+  annotations:
+    experiment.chaos-mesh.org/pause: 'true'
+  labels:
+    scenario: 'stress'
+    release: '{{ $.Release.Name }}'
+    revision: '{{ $.Release.Revision }}'
+spec:
+  scheduler:
+    cron: '@every 30s'
+  duration: '10s'
+  action: loss
+  direction: to
+  externalTargets:
+    - 'eh-stress-{{ .Release.Name }}-{{ .Release.Revision}}.servicebus.windows.net'
```

Comment:
question to Ben: where is this `eh-stress-{{ .Release.Name }}-{{ .Release.Revision}}` coming from?

it is automatically generated resource? how is the name being generated?

Reply:
This is the URI that the stress test itself will try to hit. It may be simpler to use `servicebus.windows.net` instead of trying to enumerate the actual URIs. I believe that for the JS tests this is generated from the resource group name, which itself is generated using these helm release parameters (https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/servicebus/service-bus/test/stress/test-resources.json#L66).

Reply:
yea.. I was confused because I don't see how `eh-stress-{{ .Release.Name }}-{{ .Release.Revision}}` is constructed from the file `test-resources.json` ..

It's indeed easier if we could just use the domain name `servicebus.windows.net` as the target

Reply:
I think you can but we may need to test it (e.g. whether it needs to be `.servicebus.windows.net` or `*.servicebus.windows.net` or just `servicebus.windows.net`.